packages/design-system/src/components/Tooltip/Tooltip.stories.jsx

```diff
@@ -80,3 +80,17 @@ InteractiveContent.args = {
   ),
   children: 'Tooltip with interactive content',
 };
+
+export const TooltipWithCloseButton = Template.bind({});
+TooltipWithCloseButton.args = {
+  dialog: true,
+  title: (
```

Comment:
For a while I was thinking the new focus code wasn't working, but I just didn't notice the focus state. Can you add `className: 'ds-c-button',` on this Story as well?